primes.rb

```diff
@@ -1,6 +1,16 @@
+require 'prime'
+
 class Primes
-  def self.sum_to(limit = 100)
-    # TODO - add your prime number solution here...
-    "I'm working on it!"
+  def self.sum_to(num)
+    Prime.each(num).inject(&:+)
```

Comment:
Good start.... how could you code this yourself using a prime sieve? https://en.wikipedia.org/wiki/Sieve_of_Eratosthenes